Assets/package/Runtime/Prefabs/StandardPrefabClasses/PointListClusteredParticlesPrefab.cs

```diff
@@ -0,0 +1,172 @@
+﻿/*
+ * Copyright (c) 2019 Robotic Eyes GmbH software
+ *
+ * THIS CODE AND INFORMATION ARE PROVIDED "AS IS" WITHOUT WARRANTY OF ANY
+ * KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A
+ * PARTICULAR PURPOSE.
+ *
+ */
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using UnityEngine;
+using Random = UnityEngine.Random;
+using ColoredPoint = RoboticEyes.Rex.RexFileReader.Examples.ThreadedClustering.ColoredPoint;
+
+namespace RoboticEyes.Rex.RexFileReader.Examples
+{
+	public class PointListClusteredParticlesPrefab : RexPointListObject
+	{
+		[SerializeField] 
+		private bool dynamicCloudDensityEnabled;
+		[SerializeField] 
+		private int maxAllowedParticlesOnScreen = (int)10e5;
+		[SerializeField] 
+		private int averageParticlesPerCluster = (int)60e3;
+		[SerializeField] 
+		private GameObject pointListParticlesPrefab;
```

Comment:
if you specify the Class instead of GameObject, you dont need to use GetComponent() when calling Instantiate later on.